docs/locale/fr/LC_MESSAGES/userguide/parsec_workspaces.po

```diff
@@ -0,0 +1,214 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016-present, Scille SAS
+# This file is distributed under the same license as the Parsec package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2024.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Parsec 3.0.0-rc.1+dev\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-09-18 17:42+0200\n"
+"PO-Revision-Date: 2024-09-18 18:55+0200\n"
+"Last-Translator: Marcos Medrano <marcos.medrano@scille.fr>\n"
+"Language-Team: fr <dev-parsec@scille.fr>\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Generated-By: Babel 2.15.0\n"
+"X-Generator: Poedit 3.5\n"
+
+#: ../../userguide/parsec_workspaces.rst:6
+msgid "Parsec workspaces"
+msgstr "Espaces de travail Parsec"
+
+#: ../../userguide/parsec_workspaces.rst:8
+msgid "In Parsec, your data is securely stored within **workspaces**."
+msgstr ""
+"Dans Parsec, vos données sont stockées en toute sécurité dans des **espaces "
+"de travail**."
+
+#: ../../userguide/parsec_workspaces.rst:-1
+msgid "Parsec workspace"
+msgstr "Espace de travail Parsec"
+
+#: ../../userguide/parsec_workspaces.rst:14
+msgid ""
+"You can import your data into a Parsec workspace and manage your files and "
+"directories as you will do with a regular file explorer."
+msgstr ""
+"Vous pouvez importer vos données dans un espace de travail Parsec et gérer "
+"vos fichiers et répertoires comme vous le feriez avec un explorateur de "
+"fichiers classique."
+
+#: ../../userguide/parsec_workspaces.rst:-1
+msgid "Parsec file explorer"
+msgstr "Explorateur de fichiers Parsec"
+
+#: ../../userguide/parsec_workspaces.rst:21
+msgid ""
+"Workspaces are mounted by default in your system and will also appear in the "
+"file explorer as regular folders. This is convenient to copy files from and "
+"to Parsec."
+msgstr ""
+"Les espaces de travail sont montés par défaut dans votre système et "
+"apparaissent dans l'explorateur de fichiers comme des dossiers normaux. Ceci "
+"est pratique pour copier des fichiers depuis et vers Parsec."
+
+#: ../../userguide/parsec_workspaces.rst:25
+msgid ""
+"Each workspace has its own :ref:`role-based policy "
+"<doc_userguide_parsec_workspaces_roles>` for read and write access. This "
+"allows a fine-grained access control as each user can have different roles "
+"in different workspaces."
+msgstr ""
+"Chaque espace de travail a sa propre :ref:`politique basée sur les rôles "
+"<doc_userguide_parsec_workspaces_roles>` pour l'accès en lecture et en "
+"écriture. Cela permet un contrôle d'accès très fin car chaque utilisateur "
+"peut avoir différents rôles dans différents espaces de travail."
+
+#: ../../userguide/parsec_workspaces.rst:31
+msgid ""
+"When you are offline, you can still access documents provided they were "
+"synchronized by Parsec while connected. Synchronization will occur "
```

Comment:
Only uploads are automatic, downloads are not. There's a very high chance a user won't be able to open a file if they're offline.

Reply:
I prefer to leave it unchanged: offline availability of files is a key-feature for Parsec. If there are limitations, or is not fully implemented, we should create an issue to resolve it

Reply:
https://github.com/Scille/parsec-cloud/issues/8456